frontend-react/src/pages/daily/Table/ReportsTable.tsx

```diff
@@ -100,25 +119,29 @@ function ReportsTable({ sortBy }: { sortBy?: string }) {
                 },
             },
         ],
-        rows: filteredReports,
+        rows: activeFeed || [],
     };
 
+    if (loading) return <Spinner />;
+
     return (
         <>
             <div className="grid-col-12">
-                {receiverSVCs.length > 1 ? (
+                {feeds && feeds.length > 1 ? (
```

Comment:
I think `{feeds?.length ? (` works and it shorter? 
I'm fine with not changing it, too.